app/forms/hyrax/rcr_form.rb

```diff
@@ -5,5 +5,16 @@ class RcrForm < Hyrax::Forms::WorkForm
     self.model_class = ::Rcr
     self.terms += Tufts::Terms.shared_terms
     self.required_fields = [:title, :displays_in]
+    self.field_metadata_service = Tufts::MetadataService
+
+    def self.model_attributes(_)
+      attrs = super
+      attrs[:title] = Array(attrs[:title]) if attrs[:title]
+      attrs
+    end
+
+    def title
+      super.first || ""
```

Comment:
This is just for form objects, so that you get a string for editing even if the actual attribute is an array. 

Since the controllers, models, and forms for these objects are all the same at this point I think it would make sense to just have them inherit from one generic object, that way I don't have to dupe this code in all of them. What do you think? If you think that's a good idea, I'd like to do that in a separate PR.